dot/node.go

```diff
@@ -156,10 +158,12 @@ func NewNode(cfg *Config, ks *keystore.Keystore) (*Node, error) {
 
 	// TODO: Configure node based on Roles #601
 
+	syncChan := make(chan *big.Int, 128)
```

Comment:
should `syncChan`, `networkMsgSend`, and `networkMsgRec` all be appended to configuration to reduce the number of params going into `createNetworkService` and `createCoreService` or should all channels be created at the node level and passed in as parameters? See https://github.com/ChainSafe/gossamer/pull/701#discussion_r393297573

Reply:
I think it would be nicer to put them into the config so we don't need to change function sigs

Reply:
Either way works for me, if you add `syncChan` to config here then I will add `coreMsgs` and `networkMsgs` to config in #701.

Reply:
will do!

Reply:
it's a bit of a different situation here than in NewService though, this isn't passing it into NewService, it's passing it into a function that creates the core config

Reply:
and the config that createCoreService takes is the cli config not the core config

Reply:
which is why it makes more sense to me to declare all three channels at the same time when we are creating a new node and then to pass them as channels, which would mean keeping it the way you have here and the way I have in #701... I might be confused?

Reply:
ok, I figured it out, this is fine as is, I will fix in #701